quic/s2n-quic-core/src/recovery/pacing.rs

```diff
@@ -57,6 +57,12 @@ impl Pacer {
         max_datagram_size: u16,
         slow_start: bool,
     ) {
+        // low RTT networks should not using pacing since it'll take longer to wake up from
+        // a timer than it would deliver a packet
```

Comment:
```suggestion
        // low RTT networks should not be using pacing since it'll take longer to wake up from
        // a timer than it would to deliver a packet
```